src/main/resources/application.yaml

```diff
@@ -47,16 +55,21 @@ security:
   anonymousPaths:
     - "/"
     - "/health"
+    - "/health/liveness"
+    - "/actuator/**"
     - "/loggers/**"
     - "/swagger-ui.html"
     - "/swagger-resources/**"
     - "/v2/api-docs"
     - "/webjars/springfox-swagger-ui/**"
   authorisedServices:
-    - "iac"
+    - rd_professional_api
+    - rd_user_profile
 
 ### dependency configuration
 auth.provider.service.client.baseUrl: ${S2S_URL:http://127.0.0.1:8502}
-idam.s2s-auth.totp_secret: ${IA_S2S_SECRET:AAAAAAAAAAAAAAAC}
-idam.s2s-auth.microservice: ${IA_S2S_MICROSERVICE:iac}
-idam.s2s-auth.url: ${S2S_URL:http://127.0.0.1:8502}
+idam:
```

Comment:
Ok, Please document this somewhere if you've not done so already as you are making infrastructure changes. And share with the rest of the team.